pkg/config/config.go

```diff
@@ -620,12 +620,13 @@ func (c *Config) CommittedParams() committed.Params {
 	}
 }
 
-func (c *Config) IsAuthUISimplified() bool {
-	return c.Auth.UIConfig.RBAC == "simplified"
-}
+const (
+	AuthRBACSimplified = "simplified"
+	AuthRBACExternal   = "external"
+)
 
-func (c *Config) IsAuthTypeAPI() bool {
-	return c.Auth.API.Endpoint != ""
+func (c *Config) IsAuthUISimplified() bool {
+	return c.Auth.UIConfig.RBAC == AuthRBACSimplified
```

Comment:
I believe we will need _both_ calls: eventually we will want to support ACLs on external auth servers.  Adding _another_ function and asking the right question at each step strengthens this.  Switching to a function that answers a different question needlessly makes this move harder.